### PR TITLE
Add OSS logo, used to indicate which items are open-source

### LINKED ISSUE #10: Indication if the item is open-source
I thought it would be nice to have some kind of indication if a listed item is OSS (which is very important to some users).

I've put up a [branch](https://github.com/williamboman/awesome-osx/tree/add-OSS-logo) where open-source items are flagged with the [OSI](http://opensource.org/) logo (the choice of logo is purely out of laziness). Thoughts?

### PR DESCRIPTION
**Pull request by williamboman**
Closes #10.

[Live preview](https://github.com/williamboman/awesome-osx/tree/add-OSS-logo)

### CODE REVIEW

**Comment by iCHAIT** [comment]
Sweet :smiley:, will merge this once you fix the merge conflicts.

**Comment by williamboman** [comment]
@iCHAIT Done

File: README.md [hunk 1]
```diff
@@ -21,34 +22,34 @@
...
 ### Editors
 
-- [Atom](https://atom.io/) - The hackable text editor.
-- [Brackets](http://brackets.io/) - A modern, open source text editor that understands web design.
-- [Macvim](https://github.com/b4winckler/macvim) - Vim, the text editor.
+- [Atom](https://atom.io/) - The hackable text editor. ![Open-Source Software](media/oss.png)
+- [Brackets](http://brackets.io/) - A modern text editor that understands web design. ![Open-Source Software](media/oss.png)
+- [Macvim](https://github.com/b4winckler/macvim) - Vim, the text editor. ![Open-Source Software](media/oss.png)
 - [Sublime Text 3](http://www.sublimetext.com/) - The sophisticated text editor.
 - [Textmate](https://macromates.com/) - A graphical text editor.
```
Comment (iCHAIT) [inline README.md]: Textmate is Open Source - https://github.com/textmate/textmate

Comment (williamboman) [inline README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -72,31 +73,31 @@
 - [ClipMenu](http://www.clipmenu.com/) - ClipBoard History Manager.
```
Comment (iCHAIT) [inline README.md]: ClipMenu too - https://github.com/naotaka/ClipMenu

Comment (williamboman) [inline README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -105,13 +106,13 @@
 ### Others
 
 - [Catch](http://www.giorgiocalderolla.com/index.html#catch) - The easiest way to use ShowRSS.
```
Comment (iCHAIT) [inline README.md]: Catch also - https://github.com/mipstian/catch

File: README.md [hunk 4]
```diff
@@ -105,13 +106,13 @@
...
-- [gInbox](https://github.com/chenasraf/gInbox) - gInbox is a wrapper made for Mac around Inbox by Gmail.
+- [gInbox](https://github.com/chenasraf/gInbox) - gInbox is a wrapper made for Mac around Inbox by Gmail. ![Open-Source Software](media/oss.png)
 - [Hacky](http://www.hackyapp.com/) - Hacky provides browsing Hacker News in a clean and minimalistic way.
```
Comment (iCHAIT) [inline README.md]: This too - https://github.com/eliaskg/Hacky

File: README.md (hunk 3 above)
Comment (williamboman) [inline README.md]: fixed

File: README.md (hunk 4 above)
Comment (williamboman) [inline README.md]: fixed

**Comment by iCHAIT** [comment]
Looks good to me :+1:

**Comment by iCHAIT** [comment]
Thanks a ton @williamboman , this was really awesome :smile:

**Pull request merged**